### ISSUE TITLE
VLC blanks out when creating screenshot

### ISSUE THREAD

**Issue by jat255** [flameshot-org/flameshot#515]
**Flameshot version**
```
Flameshot v0.6.0
Compiled with Qt 5.12.1
```

**VLC version**
```
LC media player 3.0.6 Vetinari (revision 3.0.6-0-g5803e85f73)
VLC version 3.0.6 Vetinari (3.0.6-0-g5803e85f73)
Compiled by builduser on arch (Apr  1 2019 20:37:10)
Compiler: gcc version 8.2.1 20181127 (GCC)
```

**Describe the bug**
When collecting a screenshot using `flameshot gui` while a VLC window is visible (in this case, an RTSP stream), the VLC window goes black, and a screenshot of the running video cannot be collected.

**To Reproduce**
1) Open a VLC RTSP stream (or probably any other video, but haven't tested)
1) Collect a screenshot using `flameshot gui`

**Expected behavior**
The video feed would not blackout, and I'd be able to take a screenshot of a running video in VLC

**System Information**
```
Operating System: Arch Linux 
KDE Plasma Version: 5.15.4
KDE Frameworks Version: 5.56.0
Qt Version: 5.12.2
Kernel Version: 5.0.7-arch1-1-ARCH
OS Type: 64-bit
Processors: 8 × Intel® Core™ i7-8650U CPU @ 1.90GHz
Memory: 15.4 GiB of RAM
```

**Comment by ianwijma** [comment]
I could be wrong, but this could be some copy write protection of VLC. But I could be wrong.

**Comment by borgmanJeremy** [comment]
This is resolved for me on X11 on version 0.8. Could you please retest?
